### PR TITLE
Form

### PR DESCRIPTION
**Pull request by Lockless404**
Completed the form page with my coding partner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ To get a local copy up and running follow these simple example steps.
 👤 Hamza Bin Ali Tarar
 
 - GitHub: [@Lockless404](https://github.com/lockless404)
```
Comment (dasileker) [inline README.md:41]: - [ ] Please add  to your readme file how you can download the project and run it locally.

File: portfolio.html [hunk 2]
```diff
@@ -135,5 +152,26 @@ <h1 class="AboutHead">About me</h1>
...
     </ul>
   </section>
+  <section class="formPage">
+    <h1 class="about-me">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form action="https://formspree.io/f/xvodlvrj" method="POST">
+      <input type="text" name="name" id="name" placeholder="Full name" class="input-name"> <br>
+      <input type="email" id="email" placeholder="Email address" class="input-email"><br>
+      <textarea name="text">Enter text here</textarea>
```
Comment (dasileker) [inline portfolio.html:162]: - [ ] Please make the all the input fields **required** as the requirement says.

llll

File: portfolio.css [hunk 3]
```diff
@@ -453,9 +453,9 @@ h3 {
   background-image: url(images/Frame.png);
   background-repeat: no-repeat;
   background-attachment: scroll;
-  background-position: 83% 32%;
+  background-position: 83% 28%;
```
Comment (dasileker) [inline portfolio.css:456]: - [ ] Your CSS is not correctly  working for the mobile view see screenshot.

- Your project 👇.
jkjkjkj

- Original project 👇

jjj

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Lockless404,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- Please make your **PR** more detailed to make the CCR know what you did.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css (hunk 3 above)
Comment (carlos-ssh) [reply portfolio.css:464]: Kindly work on the changes requested by your last reviewer. 
Stills missing the design provided on Figma
Screen Shot 2021-09-21 at 11 06 24
Screen Shot 2021-09-21 at 11 06 40
.

Before to request another code review ensure to have all the features requested on your project requirements.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Lockless404 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css (hunk 3 above)
Comment (Lockless404) [reply portfolio.css:464]: Hello. I understand the problem with the navbar, I increased the width to the size of the body including the margin of the body. But I do not understand the second problem (the second picture). This is how it looks on my pc.
Review problem
[](url)

**Review by mke2111 — APPROVED** [review]
### Hi @Lockless404, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**